libcodechecker/analyze/analyzer.py

```diff
@@ -92,7 +92,11 @@ def __get_skip_handler(args):
     try:
         if args.skipfile:
             LOG.debug_analyzer("Creating skiplist handler.")
-            return skiplist_handler.SkipListHandler(args.skipfile)
+            skip_handler = None
```

Comment:
It can be written shorter:
```py
with open(args.skipfile) as skip_file:
    return skiplist_handler.SkipListHandler(skip_file.read())
```
Please also add a documentation for this function.